tests/infra/base_tester.py

```diff
@@ -32,10 +32,12 @@ def __init__(
     ) -> None:
         self._comparison_config = comparison_config
 
-    @staticmethod
-    def _compile(executable: Callable) -> Callable:
+    def _compile(self, executable: Callable) -> Callable:
         """Sets up `executable` for just-in-time compile."""
-        return jax.jit(executable)
+        return jax.jit(executable, static_argnames=self._get_static_argnames())
+
+    def _get_static_argnames(self) -> Sequence[str]:
+        return []
```

Comment:
As said above, move this to `ModelTester` and provide a docstring by pulling out a sentence or two from `jax.jit` [docs](https://jax.readthedocs.io/en/latest/_autosummary/jax.jit.html).